content/java/concepts/calendar/terms/clone/clone.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: '.clone()'
+Description: 'Returns a copy of a Calendar object.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Calendar'
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.clone()`** method of the `Calendar` class is used to return a copy of a Calendar object.
+
+## Syntax
+
+```pseudo
+result = myCalendar.clone()
+```
+
+Returns a copy of the `myCalendar` `Calendar` object.
+
+## Example
+
+This example creates a calendar, displays it, clones it and displays its clone.
+
+```java
+import java.util.Calendar;
+  
+public class CalendarClassDemo {
+    public static void main(String args[])
+    {
+        // Create a calendar object
+        Calendar original_calendar = new GregorianCalendar(1971, 08, 13);
+  
+        // Print original calendar
+        System.out.println(original_calendar.getTime());
+  
+        // Cloning the original
+        Calendar copy_calendar = (Calendar) original_calendar.clone();
+  
+        // Print copy calendar
+        System.out.println(copy_calendar.getTime());
+    }
+}
```

Comment:
it looks as though this example was sourced from -
https://www.geeksforgeeks.org/calendar-clone-method-in-java-with-examples/?ref=ml_lbp

Technically, this is not plagiarism, but I would suggest in the future that you create your own examples/entries instead of referencing other content and making small changes. 

Also note docs examples must use spaces (2), not tabs